Update TestEval test case "regex"

TestEval "regex" does not work. We should fix or remove.

```
    eval_test.go:167: Eval() got error = failed to evaluate: invalid operation: int(string) (1:14)
         | object.image.find('v[0-9]+.[0-9]+.[0-9]*$')
         | .............^, wantErr false
    --- FAIL: TestEval/regex (0.00s)
```

--- a/eval/eval_test.go
+++ b/eval/eval_test.go
@@ -68,9 +68,8 @@ func TestEval(t *testing.T) {
 		},
 		{
 			name: "regex",
-			exp:  "object.image.find('v[0-9]+.[0-9]+.[0-9]*$')",
-			want: "v0.0.0",
-			skip: true, // https://github.com/polds/expr-playground/issues/4
+			exp:  "object.image matches 'v[0-9]+.[0-9]+.[0-9]*$'",
+			want: true,
 		},
 		{
 			name: "list",

